tests/deepsparse/pipelines/test_pipeline.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
```

Comment:
let's remove now that we have mock pipeline?

Reply:
Okay yeah - I am also going make the split and join a static method, that will also be easier to test so we don't have to create an object

Reply:
Removed in the latest push